src/Entries/SettingsEntries.php

```diff
@@ -161,7 +162,7 @@ public function getSettingsData(string $formId): array
 										'component' => 'checkbox',
 										'checkboxLabel' => \__('Store entries in database', 'eightshift-forms'),
 										// translators: %s is replaced with the form entries page URL.
-										'checkboxHelp' => $isUsed ? \sprintf(\__("You can find all form entries <a href='%s' rel='noopener noreferrer' target='_blank'>here</a>.", 'eightshift-forms'), UtilsGeneralHelper::getFormsEntriesPageUrl($formId)) : '',
+										'checkboxHelp' => $isUsed ? \sprintf(\__("You can find all form entries <a href='%s' rel='noopener noreferrer' target='_blank'>here</a>.", 'eightshift-forms'), UtilsGeneralHelper::getListingPageUrl(UtilsConfig::SLUG_ADMIN_LISTING_ENTRIES, $formId)) : '',
```

Comment:
```suggestion
										'checkboxHelp' => $isUsed ?
										\sprintf(
											\__("You can find all form entries <a href='%s' rel='noopener noreferrer' target='_blank'>here</a>.", 'eightshift-forms'),
											\esc_url(UtilsGeneralHelper::getListingPageUrl(UtilsConfig::SLUG_ADMIN_LISTING_ENTRIES, $formId))
											) :
											'',
```

Not sure which convention is used when writing complex ternaries (ternary operator at the end of the line or the beginning of the line), but this makes it a bit more readable.

Also, not sure if `esc_url()` is needed or not.